src/common/entities/task.entity.ts

```diff
@@ -0,0 +1,28 @@
+import { Entity, PrimaryKey, Property, Enum } from "@mikro-orm/core";
+
+export enum EPriority {
+  High = "High",
+  Medium = "Medium",
+  Low = "Low",
+}
+
+@Entity()
+export class Task {
+  @PrimaryKey()
+  id!: number;
+
+  @Property()
+  title!: string;
+
+  @Property()
+  summary!: string;
+
+  @Property({ type: "date", nullable: true })
```

Comment:
```suggestion
  @Property({ type: "date", nullable: true, fieldName: "due_date" })
```

Please be explicit about the fieldName in your entity file. All Property decorator inside entity file should contain fieldName for that attribute.